.ci/static-checks.sh

```diff
@@ -308,6 +308,10 @@ check_docs()
 		# This prefix requires the client to be logged in to github, so ignore
 		echo "$url"|grep -q 'https://github.com/pulls' && continue
 
+		# This prefix require the client to be logged into Jenkins, so
+		# ignore
+		echo "$url"|grep -q 'http://199.204.45.34' && continue
```

Comment:
I didn't realise we did this filtering for some URLs already.
If this list gets any longer, maybe we can pull it into an array we iterate (so, yes, a loop in a loop) - will be easier to maintain then.

Reply:
Let's fix CI first and optimize it later :-)